src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -227,6 +227,8 @@ export class CategoryFacet extends Component {
   public categoryFacetSearch: CategoryFacetSearch;
   public activeCategoryValue: CategoryValue | undefined;
   public positionInQuery: number;
+  public static MAXIMUM_NUMBER_OF_VALUES_BEFORE_TRUNCATING = 15;
```

Comment:
If these values are not meant to be customizable and are just for the tests, what about using getters for better safety?

Reply:
They aren't just for the tests I think, they are used for slicing when rendering values, doesn't seem customizable, so a using a static constant seems fine here